### DISCUSSION TITLE
Template filter working but visual feedback is not updating

### DISCUSSION THREAD

**Discussion by ThomasWT** [vercel/community#3200] [Bug Report]
### Page to Investigate

https://vercel.com/new/templates

### Steps to Reproduce

**Browser version**
_Google Chrome Version 114.0.5735.199 (Official version) (64-bit)_

1. Open a filter category.
2. Check a filter
3. Close filter category
4. Open filter category
5. Checked category does not provide visual feedback but filter is being applied

[verceltemplatefilter.webm](https://github.com/vercel/community/assets/1246641/dce94ef2-1bb3-4c9e-a6ed-915253e968e5)

**Comment by amyegan** [comment]
Thanks for reporting this @ThomasWT! It seems to have already resolved itself as I cannot replicate this behavior with any browser I have installed. Do you still see this problem?

**Reply by ThomasWT** [reply]
Not at a computer at the moment but I can confirm this also happens on my phone

https://github.com/vercel/community/assets/1246641/b0a248c0-3a77-440c-9550-486c31f9633b

**Reply by amyegan** [reply]
Thanks for sharing these videos. I'll forward them to our website team to help with investigating the cause
